src/Modal.js

```diff
@@ -48,6 +48,7 @@ const propTypes = {
   ]),
   backdropTransition: FadePropTypes,
   modalTransition: FadePropTypes,
+  innerRef: PropTypes.object,
```

Comment:
I missed this, but technically, this can still be a string or a function as well as the new createRef object.